esque/errors.py

```diff
@@ -93,6 +93,23 @@ class TopicDoesNotExistException(KafkaException):
     pass
 
 
+class TopicConfigNotValidException(ExceptionWithMessage):
+    def __init__(self, message):
+        self.message = message
+
+    def describe(self) -> str:
+        return self.message
+
+
+class YamaleValidationException(TopicConfigNotValidException):
```

Comment:
I'd put the inheritance the other way around since we might use Yamale for other validations as well.